src/Datadog.Trace.ClrProfiler.Managed/Datadog.Trace.ClrProfiler.Managed.csproj

```diff
@@ -2,12 +2,8 @@
 
   <PropertyGroup>
     <RootNamespace>Datadog.Trace.ClrProfiler</RootNamespace>
-
-    <!-- NuGet -->
     <Version>1.10.0</Version>
-    <Title>Datadog APM - ClrProfiler</Title>
-    <Description>Automatic instrumentation library used by .NET Tracer for Datadog APM. Requires separate native library that implements the CLR Profiling API.</Description>
-    <Authors>lucas.pimentel.datadog;colinhigginsdatadog;zachmontoyadd</Authors>
+    <IsPackable>false</IsPackable>
```

Comment:
```suggestion
    <Title>Datadog APM - ClrProfiler</Title>
    <Description>Automatic instrumentation library used by .NET Tracer for Datadog APM. Requires separate native library that implements the CLR Profiling API.</Description>	
    <Authors>lucas.pimentel.datadog;colinhigginsdatadog;zachmontoyadd</Authors>

    <!-- Remove the Datadog.Trace.ClrProfiler.Managed.dll assembly from the output and hide inapplicable warning (https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5128) -->
    <IncludeBuildOutput>false</IncludeBuildOutput>
    <NoWarn>$(NoWarn);NU5128</NoWarn>
```

Reply:
The observed difference from before this PR is that the generated nuspec is identical, but there is just no lib directory with `Datadog.Trace.ClrProfiler.Managed.dll|xml` assemblies

Reply:
And then the new nuspec is no longer needed 😄

Reply:
Nice! Can I remove the `nuspec` file then?

Reply:
Nice, thanks!

Reply:
Committed the suggestion in b238d6bace17d5c6a1b07f7e30d1d065e0d37a60.
Removed the `nuspec` file in 5f9757d0023c727e41099bd210e77878084bdff6.

Reply:
The NuGet package will still contain framework assembly references (until the nuspec), but I think that's fine.
```xml
    <frameworkAssemblies>
      <frameworkAssembly assemblyName="System.Configuration" targetFramework=".NETFramework4.50.0" />
      <frameworkAssembly assemblyName="System.Net.Http" targetFramework=".NETFramework4.5" />
      <frameworkAssembly assemblyName="System.ServiceModel" targetFramework=".NETFramework4.50.0" />
      <frameworkAssembly assemblyName="System.Web" targetFramework=".NETFramework4.50.0" />
    </frameworkAssemblies>
```